scripts/dts/python-devicetree/src/devicetree/edtlib.py

```diff
@@ -837,6 +842,35 @@ def spi_cs_gpio(self):
 
         return parent_cs_lst[cs_index]
 
+    @property
+    def gpio_hogs(self):
+        "See the class docstring"
+
+        if "gpio-hog" not in self.props:
+            return []
+
+        if not self.parent or not "gpio-controller" in self.parent.props:
+            _err(f"GPIO hog {self!r} lacks parent GPIO controller node")
+
+        if not "#gpio-cells" in self.parent._node.props:
+            _err(f"GPIO hog {self!r} parent node lacks #gpio-cells")
+
+        n_cells = self.parent._node.props["#gpio-cells"].to_num()
```

Comment:
Is there a reason to look in `self.parent._node.props` instead of `self.parent.props` here?

Reply:
Yes. All properties starting with `#` are stripped from `self.parent.props` but they are present in `self.parent._node.props`.